docs/summary/summary.rst

```diff
@@ -106,3 +108,18 @@ If another user has shared their storage folder to you, it will be displayed
 here. If you accept the share request, you can view and access the shared folder
 in the Data & Storage folder. The access rights are determined by the user who has sent the share request.
 Of course, you can decline the sharing requests.
+
+Download Backend.AI Web UI App
```

Comment:
This feature supports specifying the download path using the `appDownloadUrl` option.
I think it's better to add information about that!
Related PR link: https://github.com/lablup/backend.ai-webui/pull/1503

Reply:
I think this documentation is for users/admins who just simply use Backend.AI WebUI accessing to certain IP address, or URL. Therefore they can get information in [Backend.AI WebUI README](https://github.com/lablup/backend.ai-webui#configuration), instead of reading it from the manual.